modules/cert-manager-istio-creating-issuer.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-integrating-istio.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-istio-creating-issuer_{context}"]
+= Creating the issuer for the {cert-manager-operator} with Istio-CSR
+
+Use this procedure to create the issuer for
+
+[NOTE]
+====
+Other supported issuers can be used, except for the ACME issuer, which is not supported. For more information, see "cert-manager issuer types".
+====
+
+* Create a YAML file for the issuer, for example, `issuer.yaml`, with the following example:
++
+.Example `issuer.yaml` file
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Issuer <1>
+metadata:
+  name: selfsigned
+  namespace: <istio_project_name>
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: istio-ca
+  namespace: <istio_project_name>
+spec:
+  isCA: true
+  duration: 87600h # 10 years
+  secretName: istio-ca
+  commonName: istio-ca
+  privateKey:
+	algorithm: ECDSA
+	size: 256
+  subject:
+	organizations:
+  	- cluster.local
+  	- cert-manager
+      issuerRef:
+	name: selfsigned
+	kind: Issuer
+	group: cert-manager.io
```

Comment:
This may be correct for all I know, but in the preview a lot of these lines are showing red highlighting that suggest incorrect indentation. If it's correct then leave this as is, but just wanted to point that out

Reply:
Yes, it should be formatted to comply YAML spec https://github.com/openshift/openshift-docs/pull/89608#discussion_r1993984141